var/spack/repos/builtin/packages/regcm/package.py

```diff
@@ -56,20 +63,54 @@ def flag_handler(self, name, flags):
         return (None, None, flags)
 
     def configure_args(self):
+        if self.spec.satisfies(r'@4.7.0 %pgi'):
+            raise InstallError('Intel compiler not working with this specific '
```

Comment:
Probably you meant PGI compiler instead of Intel compiler? On top of that you can do checks like this one at the end of the concretization phase by using a `conflict` directive:
```python
conflict('%pgi', when='@4.7.0', msg=error_message)
```

Reply:
Yes, you're right, it's a typo.
Thanks for the tip on `msg`, I'll use it.